rascal-lsp/src/main/java/org/rascalmpl/vscode/lsp/util/Diagnostics.java

```diff
@@ -176,19 +176,31 @@ private static void storeFixCommands(IConstructor d, Diagnostic result) {
     }
 
     public static Diagnostic translateDiagnostic(IConstructor d, ColumnMaps cm) {
-        return translateDiagnostic(d, Locations.toRange(getMessageLocation(d), cm));
+        return translateDiagnostic(d, Locations.toRange(getMessageLocation(d), cm), cm);
     }
 
-    public static Diagnostic translateDiagnostic(IConstructor d, LineColumnOffsetMap cm) {
-        return translateDiagnostic(d, Locations.toRange(getMessageLocation(d), cm));
+    public static Diagnostic translateDiagnostic(IConstructor d, LineColumnOffsetMap thisFile, ColumnMaps otherFiles) {
```

Comment:
It's a bit weird that this function takes both the `ColumnMaps` as an argument, and for 1 file (the current) it already has the caller look it up. how about we change it such that we only get a `ColumnMaps` in this function? and then it's responsible for looking it up (since it already knows about `ColumnMaps`.